specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1607,6 +1607,21 @@
         "kek": {
           "$ref": "#/definitions/JsonWebKey",
           "description": "The export key encryption Json web key. This key MUST be a RSA key that supports encryption."
+        },
+        "enc": {
+          "x-ms-client-name": "enc",
+          "type": "string",
+          "minLength": 1,
+          "description": "The encryption algorithm to use to protected the exported key material",
+          "enum": [
+            "CKM_RSA_AES_KEY_WRAP",
```

Comment:
These names would not be allowed in the SDK, nor are they consistent with existing enum value names in the swagger (many of which are already non-standard names like "A192CBCAD"). Could these be made more friendly like the other enums? For example, each value can also have an `x-ms-enum` name that is maybe something like `chmRsaAesKeyWrap` which would then be consistent across languages.

Reply:
Same comment applies below.

Reply:
The other enums do not have "friendly" values, they are the values from JSON Web Algorithms. Some have additional descriptions, others do not. The particular values use, or are derived from the mechanism names in PKCS#11.